src/uid2LocalStorageManager.ts

```diff
@@ -0,0 +1,23 @@
+import { isValidIdentity, Uid2Identity } from "./Uid2Identity";
+
+export class UID2LocalStorageManager {
+  public setValue(identity: Uid2Identity) {
+    const value = JSON.stringify(identity);
+    localStorage.setItem("identity", value);
```

Comment:
`identity` probably has a high chance of conflicting with other things - I think it'd be a good idea to choose a localstorage ID that's prefixed with UID2.

Reply:
true. would `UID2` itself also run the risk of conflicts? Or is it just too vague? Perhaps `UID2-identity`?

Reply:
Alternatively, cookies use `__uid_2`. Would it make sense to use the same name or would that be more confusing?

Reply:
We sometimes find publishers store things for UID2 as well - I think making it clear that it's stored by the SDK would be helpful in a lot of cases. Maybe something like `UID2-sdk-identity`.

Reply:
done!